app/views/catalog/_ds.html.erb

```diff
@@ -1,13 +1,7 @@
 <%
   apo = @obj.admin_policy_object
   if(params[:edit])
-    if @user &&
-      (@user.is_admin ||
-        (
-          @obj.can_manage_content?(@user.roles(apo.pid)) &&
-          @obj.can_manage_desc_metadata?(@user.roles(apo.pid))
-        )
-      )
+    if can?(:manage_content, @obj) && can?(:manage_desc_metadata, @obj)
```

Comment:
Assuming this has not lost the `is_admin` but it looks like any rspec `expect` on this will call `is_admin` twice instead of once.